docs/api/components/schemas.yaml

```diff
@@ -1201,6 +1201,16 @@ Database:
       additionalProperties:
         x-additionalPropertiesName: scopename
         $ref: '#/Scopes'
+      maxProperties: 1
+      example:
+        scopename1:
```

Comment:
I might have this just be "scopename" since a part of confusion is that there can only be one.